pkgs/development/python-modules/mypy-boto3/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, boto3
+, stdenv
+, buildPythonPackage
+, mypy-boto3-builder
+, pythonOlder
+, typing-extensions
+}:
+let
+  toUnderscore = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+  buildMypyBoto3Source = serviceName: version:
+    stdenv.mkDerivation {
+      pname = "mypy-boto3-${serviceName}-src";
+      inherit version;
+
+      nativeBuildInputs = [ (mypy-boto3-builder.override { offline = true; }) ];
+      phases = [ "buildPhase" "installPhase" ];
+
+      buildPhase = ''
+        mypy_boto3_builder output --product boto3-services --services ${serviceName} --build-version ${version} --panic
+      '';
+
+      installPhase = ''
+        cp -r output/mypy_boto3_${toUnderscore serviceName}_package/. $out
+      '';
+    };
+
+  buildMypyBoto3Package = serviceName: version:
```

Comment:
I don't think we generally do package generation like this in pythonPackages.

Reply:
Do you think that it would be better just to fetch built wheels from PyPI? I think it could be also a nice option.

Reply:
As a second thought, I think also that it is not ideal, as `mypy_boto3_builder` requires python >= 3.10 and stub packages require python >= 3.7. This way we basically cut the possibility of using these stub packages with python 3.7, 3.8, and 3.9, which is not ideal. Let me rework it